package.json

```diff
@@ -2,23 +2,23 @@
 	"name": "skyra",
 	"version": "5.2.3",
 	"description": "Multipurpose Discord Bot built on Klasa",
-	"main": "dist/src/Skyra.js",
+	"main": "dist/Skyra.js",
```

Comment:
Is there such a file? Thought everything was in TS.

Reply:
Yeah, after compiling Skyra from TS to JS, `dist/Skyra.js` exists